text_scrubber/geo/find_in_string.py

```diff
@@ -278,3 +283,20 @@ def find_region_in_string(sample: str, country_set: Optional[set] = None, match_
 
     return _find_in_string(sample, clean_region, normalize_region, blacklist, whitelist_last_resort, match_threshold,
                            match_threshold_small, threshold_small, max_tokens_to_consider, country_set)
+
+
+def _precompute_bounds_find_in_string() -> None:
+    """
+    Precompute bounds for matching for the `find_in_string` functions
+    """
+    country_params = inspect.signature(find_country_in_string).parameters
+    region_params = inspect.signature(find_region_in_string).parameters
+    city_params = inspect.signature(find_city_in_string).parameters
+    for query_size in range(1, 50):
+        for params, threshold_name in product((country_params, region_params, city_params),
+                                              ('match_threshold', 'match_threshold_small')):
+            find_levenshtein_bounds(query_size, params[threshold_name].default)
+            find_trigram_bounds(query_size, params[threshold_name].default)
+
+
+_precompute_bounds_find_in_string()
```

Comment:
I am a bit lost what happens when this called .. could you expand the the method description of `_precompute_bounds_find_in_string` and state what variable is getting updated ?